server/base/src/main/java/org/apache/accumulo/server/util/RpcWrapper.java

```diff
@@ -36,20 +42,51 @@
  * @since 1.6.1
  */
 public class RpcWrapper {
+  private static final Logger log = LoggerFactory.getLogger(RpcWrapper.class);
+
+  public static <T> T service(final T instance, @SuppressWarnings("rawtypes") final Map<String,ProcessFunction<T,? extends TBase>> processorView) {
+    // Get a handle on the isOnewayMethod and make it accessible
+    final Method isOnewayMethod;
+    try {
+      isOnewayMethod = ProcessFunction.class.getDeclaredMethod("isOneway");
```

Comment:
ugh... looking at this code make me realize how brittle this new workaround and the existing work around are.   Brittle in the sense that we are expecting very specific Thrift internal behavior and method names, which could change in Thrift at any time.  I suppose we should look into pushing changes and/or test upstream that verify some sort of expected behavior.